listenbrainz_spark/stats/user/all.py

```diff
@@ -1,31 +1,58 @@
 """ Calculate and return ALL user stats in rabbitmq compatible format
 """
 from flask import current_app
-from datetime import datetime
-from listenbrainz_spark.utils import get_listens
-from listenbrainz_spark.constants import LAST_FM_FOUNDING_YEAR
-from listenbrainz_spark import path
-from listenbrainz_spark.stats.user.utils import get_artists
-from collections import defaultdict
+import listenbrainz_spark.stats.user.artist as artist_stats
 
-def calculate():
-    now = datetime.utcnow()
-    listens_df = get_listens(from_date=datetime(LAST_FM_FOUNDING_YEAR, 1, 1), to_date=now, dest_path=path.LISTENBRAINZ_DATA_DIRECTORY)
-    current_app.logger.debug("Files fetched!")
-    table_name = 'stats_user_all'
-    listens_df.createOrReplaceTempView(table_name)
 
+def calculate():
     # calculate and put artist stats into the result
     current_app.logger.debug("Running query...")
-    artist_data = get_artists(table_name)
     messages = []
+
+    current_app.logger.debug("Calculating artist_all_time...")
```

Comment:
I think it would be better if we had different commands for the new stats you're adding. Right now, if this breaks, it'll break the all time artist calculations too, which seems risky. It's definitely better to keep them seperate.